autopilot/betweenness_centrality.go

```diff
@@ -150,10 +162,48 @@ func (bc *BetweennessCentrality) Refresh(graph ChannelGraph) error {
 		return err
 	}
 
-	// TODO: parallelize updates to centrality.
-	centrality := make([]float64, len(cache.Nodes))
+	var wg sync.WaitGroup
+	work := make(chan int)
+	partials := make(chan []float64, bc.workers)
+
+	// Each worker will compute a partial result.
+	worker := func() {
+		defer wg.Done()
+		partial := make([]float64, len(cache.Nodes))
+		for {
+			node, ok := <-work
+			if !ok {
+				break
+			}
+			betweennessCentrality(cache, node, partial)
+		}
+		partials <- partial
```

Comment:
something to note for the future—it may be desirable to have the ability to cancel the calculation, either via `context.Context` or a `quit` channel to avoid blocking on shutdown for example